requirements/base-requirements.in

```diff
@@ -81,6 +81,7 @@ python-dateutil
 python-imap
 pyphonetics
 python-magic
+pytjwt
```

Comment:
Should this be `pyjwt`?

Also, I think it is necessary to run `make requirements` to update dependency comments in the requirements files even though this is already implicitly required.

Reply:
:+1: looks good